### PR TITLE
addition of asset.type

### PR DESCRIPTION
**Pull request by pisuke**
Addition of `asset.type` as discussed in #258 and #260

### CODE REVIEW

**Comment by RitaLav** [comment]
@jgunstone @TheFridgeShaman @RitaLav 

To review and add comments

**Comment by RitaLav** [comment]
Thought for consideration:

-  Repeat that Device/Asset type (asset.type) is entirely optional anywhere where it is mentioned so that the fact that it is optional is not missed or misunderstood.

File: BDNS_Specification_naming_syntax.md
Comment (jgunstone) [inline BDNS_Specification_naming_syntax.md:1]: add to definitions in "Purpose and scope" section:
* **Device/Asset Type**: a type of device that has been selected to achieve a specific function. A device type remains fixed (provided direct replacement) if the hardware is replaced. (This is an optional field).

File: BDNS_Specification_naming_syntax.md
Comment (jgunstone) [inline BDNS_Specification_naming_syntax.md:135]: optional unique ...

**Review by jgunstone — CHANGES_REQUESTED** [review]
hi - apologies I didn't make the meeting yesterday - I was fire-fighting some server issues! This looks great to me, I've made a minor comment about incorporating Device/Asset Type in the header definintions. Many Thanks

File: BDNS_Specification_naming_syntax.md
Comment (pisuke) [reply BDNS_Specification_naming_syntax.md]: Added as suggested

File: BDNS_Specification_naming_syntax.md
Comment (pisuke) [reply BDNS_Specification_naming_syntax.md:1]: Added as suggested

**Review by TheFridgeShaman — APPROVED** [review]

**Review by RitaLav — APPROVED** [review]

**Review by RitaLav — APPROVED** [review]

**Comment by RitaLav** [comment]
@jgunstone we have reviewed and approved - if you are happy with the PR we can merge to master.

**Comment by RitaLav** [comment]
Once merged we can also do a version release.

**Review by jgunstone — APPROVED** [review]
this looks great and @pisuke incorporated the comments I had - 
sorry for holding this up - I was on holiday last week. 

thanks everyone

**Pull request merged**
